File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_55673.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest55673():
    forwarded_ip = request.headers.get('X-Forwarded-For', '')
    int(str(forwarded_ip))
    return jsonify({"result": "success"})

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest55673.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest55673 {

    @GetMapping("/BenchmarkTest55673")
    public void BenchmarkTest55673(@RequestHeader("Referer") String referer, HttpServletRequest request, HttpServletResponse response) throws Exception {
        response.sendError(500, "Internal error");
    }
}
